R/epipredict-data.R

```diff
@@ -38,10 +40,12 @@
 #'
 #' Data set on state populations, from the 2019 US Census.
 #'
-#' @format Data frame with 57 rows (including one for the United States as a
-#'   whole, plus the District of Columbia, Puerto Rico Commonwealth,
-#'   American Samoa, Guam, the U.S. Virgin Islands, and the Northern Mariana,
-#'   Islands).
+#' @format An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 57 rows and 4 columns.
```

Comment:
```suggestion
#' @format A tibble (object of class `c("tbl_df", "tbl", "data.frame")`) with 57 rows and 4 columns.
```

musing: same deals as above.  Maybe this should directly link to `tibble::tibble` (and `tibble` in `Suggests:`) to mirror the above suggestion

Reply:
and by "above", apparently I mean in the file but not on GitHub?!?  it's apparently the next comment